docs/streams/developer-guide/security.html

```diff
@@ -70,7 +70,65 @@
                 the ACL set so that the application has the permissions to create, read and write
                 <a class="reference internal" href="manage-topics.html#streams-developer-guide-topics-internal"><span class="std std-ref">internal topics</span></a>.</p>
 
-                <p>To avoid providing this permission to your application, you can create the required internal topics manually.
+            <div class="admonition">
+                <p>Starting with <a class="reference external" href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-1071%3A+Streams+Rebalance+Protocol">KIP-1071</a>(Currently, it's in the Early Access phase), additional ACLs are required for the new Streams rebalance protocol. You can enable this protocol by using the <code>group.protocol=streams</code> configuration:</p>
```

Comment:
There is a space missing. Also, it's enough for this to be included in 4.2 documentation, at that point we are not in EA anymore.

```suggestion
                <p>If the <a class="reference external" href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-1071%3A+Streams+Rebalance+Protocol">streams rebalance protocol</a> is enabled by setting <code>group.protocol=streams</code>, the following ACLs are required on the topic and group resources:</p>
```